components/classes/fsi/branch.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/branch",
```

Comment:
We should reflect the "fsi" name in the schema ID too. 
"$id": "https://ns.adobe.com/xdm/fsi/classes/branch"